src/index.ts

```diff
@@ -21,8 +21,13 @@ export interface PyroscopeConfig {
   configured: boolean
 }
 
-const INTERVAL = 10000
-const SAMPLERATE = 100
+// The Interval in which samples should be collected.
+const SAMPLING_INTERVAL_MS =  process.env['PYROSCOPE_SAMPLING_INTERVAL'] || 1000 // in milliseconds
```

Comment:
```suggestion
const SAMPLING_INTERVAL_MS =  process.env['PYROSCOPE_SAMPLING_INTERVAL'] || 10 // in milliseconds
```

I guess, `SAMPLING_INTERVAL_MS` should be 10 milliseconds which corresponds with 100Hz sample rate